query-coordinator/src/test/scala/com/socrata/querycoordinator/QueryRewriterWindowFunctionTest.scala

```diff
@@ -102,6 +102,44 @@ class QueryRewriterWindowFunctionTest extends FunSuite {
     )
   }
 
+  test("Window function should rewrite when groupby clauses match, even with an extra column in the rollup") {
```

Comment:
Tests that do not _coincidently succeed_ due to the compound rewriter exact match